shell-pipelines/sort-uniq-head-tail/script-07.sh

```diff
@@ -6,3 +6,7 @@ set -euo pipefail
 # TODO: Write a command to show how many times anyone has entered and exited.
 # It should be clear from your script's output that there have been 5 Entry events and 4 Exit events.
 # The word "Event" should not appear in your script's output.
+echo "Entry:" 
```

Comment:
This works, but isn't super general - imagine in the future a new event type got added, perhaps "Refused" - can you rewrite this so that it will work for _any_ events that happen, not just Entry and Exit?